app/views/posts/_expanded.html.erb

```diff
@@ -293,6 +293,33 @@
           </div>
         <% end %>
 
+        <% if is_question && post.children.length >= SiteSetting["TableOfContentsThreshold"] %>
+          <div class="toc" id="toc-example-toggle">
+            <button class="toc--header" data-toggle="#toc-example-toggle" data-toggle-property="class" data-toggle-value="is-active">Table of Contents</button>
```

Comment:
[**suggestion (if-minor):**](https://conventionalcomments.org/) It would be super-awesome if communities could customize the title of the TOC. For example Code Golf might want to name it "Leaderboard".

Reply:
I'm sure that would be a good thing but I'll refrain from that right now since we are keeping things at a simpler side

Reply:
👍 Makes sense. We can add a site setting for that later, too.